### ISSUE TITLE
Create OntoGPT runner

### ISSUE THREAD

**Issue by yaseminbridges** [monarch-initiative/pheval#159]

**Comment by yaseminbridges** [comment]
Very basic runner set up at: https://github.com/yaseminbridges/pheval.ontogpt

API key for open AI expected to be set up beforehand

**Comment by matentzn** [comment]
Wow great work!! Does this work at all? Is it ready to be folded as into the main pipeline?

**Comment by yaseminbridges** [comment]
Yep, this does work for me! It'd be great if @souzadevinicius could test on his machine to see if there are any problems, but I would say it is ready to be folded into the main pipeline
